### ISSUE TITLE
High DICOM and Supplement 222 Microscopy Bulk Simple Annotations Storage SOP Class

### ISSUE THREAD

**Issue by DrexlerChao** [MGHComputationalPathology/highdicom#98]
Dear Sir,

As we know, the High DICOM support
1. Creating Segmentation (SEG) images
2. Parsing Segmentation (SEG) images
3. Creating Structured Report (SR) documents
4. Parsing Structured Report (SR) documents

But we are not sure whether High DICOM can support Supplement 222 Microscopy Bulk Simple Annotations Storage SOP Class.

This Supplement to the DICOM Standard specifies a new DICOM Information Object and Storage SOP Class for storing Microscopy Bulk Simple Annotations (points, open polylines, closed polygons and simple geometric shapes without relationships), which is referred to as the Microscopy Bulk Simple Annotations IOD.

Microscopy Bulk Simple Annotations are usually created by machine algorithms from high resolution images of entire tissue sections, e.g., encoded as DICOM Whole Slide Microscopy images.

If High DICOM can support, it will be easily used after feeding data into AI.

**Comment by hackermd** [comment]
Yes, it's on the roadmap. We'll work on it once parsing of SEG and SR instances is finalized (#93).
